lib/bundler/definition.rb

```diff
@@ -267,7 +267,7 @@ def lock(file)
       File.open(file, 'wb'){|f| f.puts(contents) }
     rescue Errno::EACCES
       raise Bundler::InstallError,
-        "There was an error while trying to write to Gemfile.lock. It is likely that \n" \
+        "There was an error while trying to write to #{File.basename(file)}. It is likely that \n" \
```

Comment:
Oh I see, this is the only place where we have `File` available so why not use it? I get that.

I lean slightly more on the side of consistency - if I were to see this break in the pattern, I'd wonder why, but I'm like a 2 out of 10 on this perspective.

@indirect thoughts? For me, this is ready to ship aside from this discussion.